responsibleai/responsibleai/rai_insights/rai_insights.py

```diff
@@ -191,6 +191,10 @@ def _validate_model_analysis_input_parameters(
                 'INVALID-MODEL-WARNING: No valid model is supplied. '
                 'The explanations, error analysis and counterfactuals '
                 'may not work')
+            if serializer is not None:
+                raise UserConfigValidationException(
+                    'No valid model is specified but model '
```

Comment:
Not a big deal, but you can put that string in a constant and refer to it in the tests, rather than writing it twice. If you choose not to that's fine, too.

Reply:
There are a lot of messages like that currently. Maybe we should create an issue and move to the suggested pattern for all of them.